File: AD-HOC/PYTHON/TELA_3/1366.py
while True:
    N = int(input())

    if N == 0:
        break
    else:
        ACUM = 0

        for i in range(1, N + 1):
            T, Q = map(int, input().split())

            if Q % 2 == 0:
                ACUM += Q
            else:
                ACUM += Q - 1

        print(ACUM // 4)

File: AD-HOC/C#/TELA_3/1366.cs
using System;

class Program
{
    static void Main()
    {
        while (true)
        {
            int N = int.Parse(Console.ReadLine());

            if (N == 0)
                break;
            else
            {
                int ACUM = 0;

                for (int i = 1; i <= N; i++)
                {
                    string[] input = Console.ReadLine().Split();
                    int T = int.Parse(input[0]);
                    int Q = int.Parse(input[1]);

                    if (Q % 2 == 0)
                        ACUM += Q;
                    else
                        ACUM += Q - 1;
                }

                Console.WriteLine(ACUM / 4);
            }
        }
    }
}
